### ISSUE TITLE
1.49 release not downloading the new finetune training files

### ISSUE THREAD

**Issue by Mixerrog** [deezer/spleeter#233]
## Step to reproduce

1. Installed using `...`
2. Run as `...`
3. Got `...` error

## Output

```bash
Share what your terminal says when you run the script (as well as what you would expect).
```

## Environment

|                   |                                 |
| ----------------- | ------------------------------- |
| OS                | Windows / Linux / MacOS / other |
| Installation type | Conda / pip / other             |
| RAM available     | XGo                             |
| Hardware spec     | GPU / CPU / etc ...             |

**Comment by Mixerrog** [comment]
You stated in response to me in #232 that the new 1.49 version uses the new finetune training files but checking the  pretrained models files folder, it only downloads the original  2,4 & 5 stems training files & none of the new larger finetune files so how could the new version use these if not downloaded? Is there an issue with the new software not requesting these or are they being stored elsewhere as it appears the new version is not using the finetune training files to me?

Thanks, for your support, Roger

**Comment by Mixerrog** [comment]
Any answers to this yet?

Thanks, Roger

**Comment by MrBasssman65** [comment]
I had issues with this until I tried an experiment.  In my Spleeter folder I had the standard 2, 4 and 5 stems.  When I updated to 1.49, it didn't auto download the new finetune files for me either.  I manually downloaded the finetune models to my Spleeter folder and then tried something.  I removed the standard 4 stems folder and then used the command line to direct the program to the 4stems-16kHz finetune folder.  What happened was the program replaced the standard 4 stems folder and used that instead.  So I got to thinking that if I take out all 3 standard folders and place them on another drive for safekeeping, the program will only look for the finetune folders.  This was confirmed by me as my new separations from that point on all went to 16kHz and no new standard models were downloaded.  So if the finetune folders don't automatically download, try to do it manually and then get rid of the standard ones and make sure the 16kHz .json files are also in the correct location.  So far, that is what has worked for me.
